### ISSUE TITLE
Modernize Drops

### ISSUE THREAD

**Issue by BlackJar72** [Sunconure11/dungeon-mobs#16]
Make the mobs drops that don't require custom enchantment code use modern, data pack friendly, JSON files.

**Comment by LemADEC** [comment]
For the record, we can't modify drops using CraftTweaker2-1.12-4.1.9 either.
It feels like the mod doesn't use the recently added loot tables.

**Comment by BlackJar72** [comment]
No, it doesn't use these at all, that's the point.  All the drops are hard-coded, just as in the 1.6.4 and 1.7.10 versions.
